src/replica/replica_stub.cpp

```diff
@@ -272,6 +282,16 @@ DSN_DEFINE_uint32(
     "The interval milliseconds of replica server to syncs replica configuration with meta server");
 DSN_TAG_VARIABLE(config_sync_interval_ms, FT_MUTABLE);
 DSN_DEFINE_validator(config_sync_interval_ms, [](uint32_t value) -> bool { return value > 0; });
+DSN_DEFINE_group_validator(encrypt_data_at_rest, [](std::string &message) -> bool {
+    if (!dsn::security::FLAGS_enable_acl || !FLAGS_encrypt_data_at_rest) {
+        message = fmt::format("FLAGS_enable_acl = ({}) FLAGS_encrypt_data_at_rest = ({}),should be "
```

Comment:
Use the identify in the config file (i.e. `[security] enable_acl`) would be more readablity for Pegasus users, they don't know what does FLAGS_xxx mean.